protzilla/run.py

```diff
@@ -178,6 +178,10 @@ def current_plots(self) -> Plots | None:
     @property
     def current_outputs(self) -> Output:
         return self.steps.current_step.output
+    
+    @property
+    def current_filtered_data(self) -> Output:
```

Comment:
Oder entsprechend Line 38 in steps.py in Output() ändern
```suggestion
    def current_filtered_data(self) -> dict:
```

Reply:
Ich würde es fast eher bei Output lassen um einheitlich definiert zu haben "was für eine Art dict" es ist, da Output ja auch mainly ein dict mit schönerem Namen ist
Könnte man so erreichen:
```suggestion
    def current_filtered_data(self) -> Output:
            output = Output(self.steps.current_step.datatable_filtered_data)
        return output
```

(ooomg das ist ja cool mit den suggested changes)

Reply:
Ich würde es auf dict ändern, da es ja kein richtiger Output mehr ist, wenn die Tabelle gefiltert wird. Dann gibt es nicht zwei Outputs pro Step, sondern nur einen Output und ein gefiltertes dict.

Reply:
Klingt sinnvoll!